mne/io/edf/edf.py

```diff
@@ -334,6 +334,10 @@ def _read_header(fname, exclude):
     exclude : list of str
         Channel names to exclude. This can help when reading data with
         different sampling rates to avoid unnecessary resampling.
+    misc : list or tuple
```

Comment:
list and tuple should be treated equivalently ([Duck typing](https://en.wikipedia.org/wiki/Duck_typing)), so simply listing one of them here should suffice.